### ISSUE TITLE
Training doesn’t start with start_from for domain training

### ISSUE THREAD

**Issue by alpoktem** [OpenNMT/OpenNMT-py#1364]
Hi,

I am having a strange problem trying to do domain adaptation for an MT model in OpenNMT-py. I have previously trained a model on a large dataset. Now I want to resume training on a different smaller dataset. I first preprocessed the new dataset and obtained pt files for that. Then to resume training I make the following command:

> python train.py -data -save_model -coverage_attn -word_vec_size 512 -layers 3 -rnn_size 512 -rnn_type LSTM -encoder_type brnn -batch_size 64 -dropout 0.2 -input_feed 1 -global_attention mlp -optim adam -learning_rate 0.0001 -gpu_ranks 0 -train_steps 10000 -report_every 1 -save_checkpoint_steps 10 -valid_steps 50 -train_from

It loads the files and the old model but then just stops there. This is the [err output that I get](https://pastebin.com/uTrs70n3)

I tried taking out the `-train_from` part and it trains without any problem.

I’d appreciate any help.

**Comment by vince62s** [comment]
there is no error in your log

**Comment by alpoktem** [comment]
Exactly. It doesn't do any training either.

**Comment by guillaumekln** [comment]
`-train_steps` is absolute, not relative. So you should set it to 83600 for example.

**Comment by alpoktem** [comment]
OK!
